api/resource_releases.go

```diff
@@ -105,3 +108,31 @@ func (c *Client) GetAppReleaseNomad(ctx context.Context, appName string, id stri
 
 	return data.App.Release, nil
 }
+
+func (c *Client) CreateReleaseMachines(ctx context.Context, input CreateReleaseInput) (*CreatedRelease, error) {
+	query := `
```

Comment:
thanks for moving this here! No need to make a change; this is just an fyi...

It's ok to use the genqlient code in the api package, too. It's the newer way to do graphql queries and avoids the tedium of having to write out input/output the structs by hand. We haven't backported genqlient to all the existing gql queries that use this pattern (and likely won't backport them).

No change needed here. Next time, feel free to use genqlient though!

Reply:
Good to know! I searched for it, and found literally nowhere else using it so I figured I might as well port it over to how everything else does it.

Only now realizing that I searched for `genqclient` not `genqlient`!